.github/workflows/github-actions.yml

```
@@ -64,10 +67,13 @@ jobs:
       # 도커 이미지를 빌드하고 도커 허브에 업로드
       - name: Docker image build & push
         run: |
+          DEPLOY_VERSION=$(cat version | tr -d '\r\n')
+          echo ">>> DEPLOY_VERSION: $DEPLOY_VERSION"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)